### PR TITLE
docs: self-hosted share in slack

### PR DESCRIPTION
**Pull request by rephus**
Closes: #3307

### Description:

https://deploy-preview-4039--peaceful-bassi-cbf284.netlify.app/guides/enable-slack-selfhost/

image

### CODE REVIEW

File: docs/docs/guides/enable-slack-selfhost.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+You can select `From an app manifest` to make it easier. 
+
+<img src={SlackCreateNewApp} width="500" height="400" style={{display: "block", margin: "0 auto 20px auto"}}/>
+
+Then select the workspace you want to enable this into. Later you can `enable distribution` if you want to use a different Slack workspace.
+
+<img src={SlackSelectWorkspace} width="500" height="400" style={{display: "block", margin: "0 auto 20px auto"}}/>
+
+Then copy this manifest to allow URL unfurls in your app. Make sure you update `your Lightdash URL` in the manifest too.
```
Comment (TuringLovesDeathMetal) [inline docs/docs/guides/enable-slack-selfhost.mdx:23]:
```suggestion
Then copy this manifest to allow URL unfurls in your app. **Make sure you update `your Lightdash URL` in the manifest below** (for example, `app.lightdash.cloud` or `eu1.lightdash.cloud`).
```

**Review by TuringLovesDeathMetal — CHANGES_REQUESTED** [review]
Looks great! Just one small change for the manifest file :)

**Review by TuringLovesDeathMetal — COMMENTED** [review]
Also, can you add this doc to the sidebar here? Or another option would be to merge the two docs and just have headers like `If you're using Lightdash Cloud` then another which is `If you're self-hosting Lightdash`?

image

**Comment by rephus** [comment]
> Also, can you add this doc to the sidebar here

I thought about this, but I think it could be confusing, because if you're on self-hosting, you need to enable it first. What I did was to link to the new document inside `share in slack` , at the top.... What do you think ?

**Comment by TuringLovesDeathMetal** [comment]
@rephus oh nice! Sorry, I didn't see that there 😅 

Maybe we could use an info box or something?? Like this:

```
:::info

blah blah

:::
```

**Comment by rephus** [comment]
Yeah, much better, thanks. I was not sure how to add that block 

image

**Review by TuringLovesDeathMetal — APPROVED** [review]

**Pull request merged**